### ISSUE TITLE
Flarum is not Accessible

### ISSUE THREAD

**Issue by nsakibp** [mondediefr/docker-flarum#111]
How to access the site after successful installation?
It's not resolving to the port. Even after Reverse proxy setup.

Can you please put some light on it?
Thanks in advance

**Comment by Hydrog3n** [comment]
Hello @nsakibp Can you given us more information ?

**Comment by nsakibp** [comment]
Hi
The problem is solved. The port will be "8888:8888" instead of "80:8888"

I have another question.
"docker exec -ti flarum extension require some/extension"

This command installs extension to Flarum. 
If I have more than one Flarum stack on docker, on which instance the extension will be installed?

Is there any mapping for that?
Please let me know

Thanks in advance

**Comment by Hydrog3n** [comment]
@nsakibp Hello, 
Normally if both have the same mounting point extensions should be ok. 
You can also use the file for list of extentions

**Comment by nsakibp** [comment]
Thanks a lot again for your prompt response

Can you please tell me what you mean by mounting point?
Is it Docker volume?

**Comment by Hydrog3n** [comment]
Yes it's the mounting point. At mondedie.fr we was using with two container with scale in docker-compose was working we stop because not really usefull but it was working well

**Comment by nsakibp** [comment]
Thanks a lot again.

Did you mean, If I use Flarum for two different projects on two different
containers,
Then the extension install command will install an extension on those two
different mount points simultaneously?

Thanks

**Comment by Hydrog3n** [comment]
Ah ! So no you can't have link like I sayed. 

You can create a file call `file` in extensions with the list of all your extensions so no need to add it each time

**Comment by neoranger** [comment]
> Hi The problem is solved. The port will be "8888:8888" instead of "80:8888"
> 
> I have another question. "docker exec -ti flarum extension require some/extension"
> 
> This command installs extension to Flarum. If I have more than one Flarum stack on docker, on which instance the extension will be installed?
> 
> Is there any mapping for that? Please let me know
> 
> Thanks in advance


To it's not work. With the 8888:8888 port i get this:
![image](https://github.com/mondediefr/docker-flarum/assets/5490758/bc177d26-c176-43cb-ad1e-7502985dfc1a)
